app/mailboxes/support_mailbox.rb

```diff
@@ -12,6 +12,14 @@ def process
     # prevent loop from chatwoot notification emails
     return if notification_email_from_chatwoot?
 
+    # return if email doesn't have a valid sender
+    # This can happen in cases like bounce emails for invalid contact email address
+    # TODO: Handle the bounce seperately and mark the contact as invalid
+    # we are checking for @ since the returned value could be "\"\"" for some email clients
+    return unless original_sender_email.include?('@')
+
+    Rails.logger.info "Processing email from #{original_sender_email} to #{mail.to} with subject #{mail.subject}"
```

Comment:
Can we add this log above?

Reply:
With this implementation, we would miss the emails with invalid to. Also I would suggest logging the email id.